### ISSUE TITLE
Selection seems to break after doing a Find in Project

### ISSUE THREAD

**Issue by Gruntfuggly** [bigfive/atom-sublime-select#81]
This package works fine for a while but then seems to go wrong. The highight disappears and the selected range seems to be in the wrong place? 

It was driving me nuts, but I eventually worked out that it breaks after doing a Find in Project.

**Comment by bigfive** [comment]
Hmm, I cant seem to reproduce this. Can you give me any more info on the steps required?

**Comment by alexhicks** [comment]
@bigfive It's reproducible when you `Split Right` then go back to the original pane and try doing to a range select. I was looking into it last year and the origin of the pane seems to no longer be zero in the original pane.

![select-issue](https://cloud.githubusercontent.com/assets/7707452/12344436/1a9a0f9e-bb91-11e5-9d3e-8d68219e4649.gif)

**Comment by bigfive** [comment]
Yep. I can reproduce the issue with Split panes. I'll be looking into that one. I assumed this one was different.
